### DISCUSSION TITLE
Salary and deductions screens not working yet

### DISCUSSION THREAD

**Discussion by einhverfr** [ledgersmb/LedgerSMB#5832] [Ideas]
https://sourceforge.net/p/ledger-smb/bugs/1236/

Need to be able to view and add deductions and salary elements.  The reports are not working either.  So this will require a bit of effort (so moving it here)

**Comment by einhverfr** [comment]
see also https://sourceforge.net/p/ledger-smb/bugs/1375/

**Comment by ehuelsmann** [comment]
Moving to 1.6 -- apparently, there's no huge need for payroll as I have had no complaints ever (personally).

**Comment by ehuelsmann** [comment]
Let's keep this blocking for 1.6; having this broken for _at least_ two minor versions _does_ look sloppy.

**Comment by sbts** [comment]
As discussed in chat adding a config option to enable / disable (hide) payroll is probably desirable at this point as the functionality is not complete and we don't currently have anyone with an urgent enough need to raise it's priority. 

As long as it's disabled by default and the means to enable it is easily discovered and warns that the functionality is incomplete I'd suggest this is done before 1.6-rc0

**Comment by ehuelsmann** [comment]
With a due_date of 2.0, shouldn't we disable those screens by default only to enable them with a setting in defaults?

**Comment by ehuelsmann** [comment]
As of 1.8, the wages&deductions screen is hidden by default.
